### DISCUSSION TITLE
Helping translations

### DISCUSSION THREAD

**Discussion by jwchen119** [LMS-Community/slimserver#904] [General]
Hi Michael,

I would like to know if the official way to impelement a new language "zh-TW" into slimserver is to submit a PR?
If so, is it correct to add the translations in any string.txt files?

**Comment by mherger** [comment]
You'll have to add them in _every_ `strings.txt` file you can find (at the root, and in `Slim/Plugin/*`). It's a lot... I'd start with the main file in the root. Please note that you'll have to use `ZH_TW` (all uppercase, underscore) as the identifier. And make sure you use tabs to indent and separate token from value. And please add them at the bottom of each block, below `ZH_CN`.

Add a few values, test locally, continue. It'll be a lot of work... Thanks!

**Reply by jwchen119** [reply]
I found not all of the strings are included ZH_CN, which means the ZH_TW should gose on the very buttom, even without ZH_CN, right?
